examples/java-matter-controller/java/src/com/matter/controller/commands/common/RealResult.java

```diff
@@ -0,0 +1,42 @@
+/*
+ *   Copyright (c) 2023 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+package com.matter.controller.commands.common;
+
+public class RealResult implements Result {
```

Comment:
please add some factory methods. The usage of `r = new RealResult(); r.setSucces(); use r` is ugly. It would be easier to read `use RealResult::Success()` or similar. So I expect:

```java

public static RealResult Success() {
  // ...
}

public static RealResult Error(String error) {
  // ...
}
```

Reply:
Done, thanks